lib/src/legacy/value/index.ts

```diff
@@ -17,3 +17,6 @@ export const Map = LegacyMap;
 export const Null = SassNull;
 export const Number = LegacyNumber;
 export const String = LegacyString;
+
+// For the `sass.types.Error` object, we just re-export the native Error class.
+export const Error = global.Error;
```

Comment:
I think this is missing some fields from the legacy error object.

Reply:
No, this isn't the `LegacyError` class—this is the `sass.types.Error` class that's only used for async custom functions to communicate errors back to the caller. I'm not sure why the tests are failing, they pass locally for me.

Reply:
Oh I see, I think we just need to update the compiler version.